metadata/routes.py

```diff
@@ -69,6 +69,24 @@ def get_credentials():
         response.status = e.status
         return {'error': {'message': e.message}}
 
+@route('/latest/dynamic/instance-identity/document')
+def get_identity_document():
+    return {
+        'privateIp' : '127.0.0.1',
+        'devpayProductCodes' : None,
+        'availabilityZone' : 'us-east-1a',
+        'version' : '2010-08-31',
+        'instanceId' : 'i-12345678',
+        'billingProducts' : None,
+        'instanceType' : 't2.nano',
```

Comment:
I called it on a t2.nano, ergo it returned t2.nano.  Agree, it doesn't matter, what matters is that this endpoint returns something.